system-test/datastore.ts

```diff
@@ -1323,18 +1323,61 @@ describe('Datastore', () => {
     const gcs = new Storage();
     const bucket = gcs.bucket('nodejs-datastore-system-tests');
 
+    const setupForDelay = (test: Mocha.Context) => {
+      if (!test.currentAttempt || typeof test.currentAttempt !== 'number') {
+        test.currentAttempt = 0;
+      }
+      test.currentAttempt++;
+    };
     const delay = async (test: Mocha.Context) => {
-      const retries = test.currentRetry();
-      if (retries === 0) return; // no retry on the first failure.
+      const retries = test.currentAttempt - 1;
+      if (retries === 0) return; // no retry on the first attempt.
       // see: https://cloud.google.com/storage/docs/exponential-backoff:
       const ms = Math.pow(2, retries) * 500 + Math.random() * 1000;
       return new Promise(done => {
-        console.info(`retrying "${test.title}" in ${ms}ms`);
+        console.info(`retrying "${test.test?.title}" in ${ms}ms`);
         setTimeout(done, ms);
       });
     };
 
+    describe('running tests against the delay function', () => {
+      let consoleInfoFunction: (message: string) => void;
+      let infoLogCount = 0;
+
+      before(async () => {
+        consoleInfoFunction = console.info;
+      });
+
+      it('should be sure that the delay function emits console info messages', async function () {
+        // Override console.info to track the number of times it is called.
+        console.info = consoleInfoFunction;
```

Comment:
Why is this required if the next line is overriding this assignment?

Reply:
I think this was a typo which needs to be removed.